### PR TITLE
[[Rewrite Header Widget]] New header widget added and old one removed

### PR DESCRIPTION
**Pull request by livecodegeorgia**

### CODE REVIEW

File: extensions/widgets/header/header.lcb [hunk 1]
```diff
@@ -68,551 +94,1036 @@ Sets the header actions display style of the control.
...
+
+      variable tActionCount as Integer
+      variable tNumActions as Integer
+
+      put the number of elements in element 1 of mHeaderActions into tNumActions
+
+      repeat with tActionCount from 1 up to tNumActions
+
+         if tActionCount is tSelectedUp-2 then
+            post "headerAction" with mHeaderActions[tActionCount]["name"]
```
Comment (livecodeali) [inline extensions/widgets/header/header.lcb]: The 'with' argument of post has to be a list (even if 1 element), so this should be 

   post "headerAction" with [mHeaderActions[tActionCount]["name"]]

File: extensions/widgets/header/header.lcb [hunk 2]
```diff
@@ -68,551 +94,1036 @@ Sets the header actions display style of the control.
...
+   variable tSize as Number
+   variable tFont as String
+
+   if pType is "title" then
+      put (my height)*mTitleRatio into tSize
+      put "Helvetica Neue Regular" into tFont
+
+   else if pType is "subtitle" then
+      put (my height)*kSubtitleRatio into tSize
+      put "Helvetice Neue Regular" into tFont
```
Comment (livecodeali) [inline extensions/widgets/header/header.lcb]: Typo here :-)

**Comment by livecodeali** [comment]
@livecode-vulcan review ok 5e33e5e

**Comment by livecode-vulcan** [comment]
💙 review by @livecodeali ok 5e33e5ee61da3f9c256016b6d628a00c5c896262

**Pull request merged**